tern/report/report.py

```diff
@@ -129,11 +129,13 @@ def load_full_image(image_tag_string):
 def generate_report(args, *images):
     '''Generate a report based on the command line options'''
     if args.report_format:
-        return generate_format(images, args.report_format)
-    return generate_format(images, 'default')
+        if args.report_format == 'html':
```

Comment:
We don't need to specifically check for html format to generate the report this way. This is handled by the plugin manager in the `generate_format` function.